oshi-core/src/test/java/oshi/PerfMonGpuEngineTest.java

```diff
@@ -0,0 +1,62 @@
+package oshi;
+
+import oshi.util.platform.windows.PerfCounterQuery;
+import oshi.util.platform.windows.PerfCounterWildcardQuery;
+import oshi.util.tuples.Pair;
+
+import java.util.List;
+import java.util.Map;
+
+public class PerfMonGpuEngineTest {
+
+    public enum GpuEngineProperty implements PerfCounterWildcardQuery.PdhCounterWildcardProperty {
+        NAME(PerfCounterQuery.NOT_TOTAL_INSTANCE),
+        RUNNING_TIME("Running Time"),
+        UTILIZATION("Utilization Percentage");
+
+        private final String counter;
+
+        GpuEngineProperty(String counter) {
+            this.counter = counter;
+        }
+
+        @Override
+        public String getCounter() {
+            return counter;
+        }
+    }
+
+    // hmmmmm individual instance dont look right......
+    public enum GpuEngineProperty2 implements PerfCounterQuery.PdhCounterProperty {
```

Comment:
You're mixing types.  Since you're doing a wildcard enum you don't want the PdhCounterProperty, you want PdhCounterWildcardProperty.